apt/risk/data_assessment/dataset_attack_result.py

```diff
@@ -0,0 +1,24 @@
+from dataclasses import dataclass, field
+
+import numpy as np
+
+
+@dataclass
+class DatasetAttackScore:
```

Comment:
I think it may be helpful to have the score as part of the base class, this can enable generic treatment of different types of scores.